include/emitc/emitc_tosa.h

```diff
@@ -49,6 +49,72 @@ inline Src add(Src x, Src y) {
   return emitc::add<Src>(x, y);
 }
 
+// MulOp
+template <typename Src>
+inline Src mul(Src x, Src y, int32_t shift) {
+  assert(shift == 0); // shift has to be 0 if datatype is not int32_t
+  return emitc::mul(x, y);
+}
+
+template <>
+inline Tensor0D<int32_t> mul(Tensor0D<int32_t> x, Tensor0D<int32_t> y,
+                             int32_t shift) {
+  // Taken from
+  // https://git.mlplatform.org/tosa/reference_model.git/tree/reference_model/src/ops/ewise_binary.cc?id=df8626976df6c779bb30df9c5ceef689462109c0#n436
```

Comment:
To which portion of you code does this apply? We may have to mark the taken over parts more clearly. Further, have you checked the license?